### ISSUE TITLE
If header version is malformed it is only logged but not reported back.

### ISSUE THREAD

**Issue by ravn** [veraPDF/veraPDF-parser#290]
While writing tests for file descriptor leaks I noticed that a stacktrace was written to the log.

The code in PDFParser.java is

```
        float headerVersion = 1.4f;
        try {
            String[] headerParts = header.split("-");
            if (headerParts.length == 2) {
                headerVersion = Float.parseFloat(headerParts[1]);
            }
        } catch (NumberFormatException e) {
            LOGGER.log(Level.WARNING, "Can't parse the header version.", e);
        }

        result.setVersion(headerVersion);
        checkComment(result);
```

and is triggered by one of the files in the corpus (most likely veraPDF-corpus/PDF_A-1b/6.1 File structure/6.1.2 File header/veraPDF test suite 6-1-2-t01-fail-b.pdf).  As I read the code, this condition is only logged and then silently defaulted to 1.4, so this does not show in the XML reported to the user.

**Comment by bdoubrov** [comment]
Yes, I confirm that veraPDF defaults the header version to 1.4 in case it can't parse the header. It was implemented this way to be consistent with PDFBox implementation. 

I also note that the file header and the version of the PDF document might differ, if the Catalog dictionary contains /Version entry, which takes precedence over the header version.

What we can indeed do is to include the original header string to the feature XML report as is. Is this what you would want to see?

**Comment by ravn** [comment]
I would like for the logger statement to be removed.  I am not a decision maker, so I cannot say how the information should be propagated to the actual decision maker (about the severeness of this) but I think they should be told instead of just silently ignoring it.
